src/EditorFeatures/Core/Classification/Syntactic/SyntacticClassificationTaggerProvider.TagComputer.cs

```diff
@@ -272,15 +282,18 @@ private async ValueTask ProcessChangesAsync(ImmutableSegmentedList<ITextSnapshot
                 Contract.ThrowIfTrue(snapshots.IsDefault || snapshots.IsEmpty);
                 var currentSnapshot = GetLatest(snapshots);
 
-                var classificationService = TryGetClassificationService(currentSnapshot);
-                if (classificationService == null)
+                if (TryGetClassificationService(currentSnapshot) is not (var solutionServices, var classificationService))
                     return;
 
                 var currentDocument = currentSnapshot.GetOpenDocumentInCurrentContextWithChanges();
                 if (currentDocument == null)
```

Comment:
*aside*, this whole pattern is wonky.  we get the document or not... so we dont need the helper for getting the classificationservice/solution-service.  We could always just get that from currentDocument.  Would simplify a lot it seems.